app/src/main/java/io/apicurio/registry/storage/impl/sql/CommonSqlStatements.java

```diff
@@ -992,12 +992,12 @@ public String selectTenantIdsByConfigModifiedOn() {
 
     @Override
     public String deleteAllReferences() {
-        return "DELETE FROM artifactreferences ar WHERE ar.tenantId = ?";
+        return "DELETE FROM artifactreferences WHERE tenantId = ?";
     }
 
     @Override
     public String deleteOrphanedReferences() {
-        return "DELETE FROM artifactreferences ar WHERE NOT EXISTS (SELECT 1 FROM versions v WHERE v.contentId = ar.contentId AND v.tenantId = ar.tenantId)";
+        return "DELETE FROM artifactreferences WHERE NOT EXISTS (SELECT 1 FROM versions v WHERE v.contentId = contentId AND v.tenantId = tenantId)";
```

Comment:
Here I'm puzzled, I thought that we should assume no support for aliases, but here they are still used to disambiguate content.
If aliases are supported, can we keep(most of) the original implementation in this file?
If they are not, would you mind pointing out to documentation regarding this limitation?

Thanks in advance! 🙏

Reply:
@andreaTP the DELETE syntax differs between [sqlserver](https://learn.microsoft.com/en-us/sql/t-sql/statements/delete-transact-sql?view=sql-server-ver15#syntax) and [postgres](https://www.postgresql.org/docs/current/sql-delete.html) with the positioning of aliases in the query. In order to keep the common queries I just dropped the alias.
For SELECT it's compatible between both, so I kept it as it is.